src/app/shared/image/image-loader.component.ts

```diff
@@ -4,18 +4,22 @@ import { Component, Input, ElementRef } from '@angular/core';
   selector: 'image-loader',
   styleUrls: ['image-loader.component.css'],
   template: `
-    <img *ngIf="src" [src]="src" class="loading" (load)="onLoad()"/>
+    <img *ngIf="src" [src]="src" class="loading" (load)="setBackgroundImage(src)" (error)="setBackgroundImage(placeholderError)"/>
+    <img *ngIf="!src" [src]="placeholderImage" class="loading" (load)="setBackgroundImage(placeholderImage)" (error)="setBackgroundImage(placeholderError)"/>
```

Comment:
So thinking about the order of events here ...

(1) Ask CMS for the `prx:image`
(2) Component initialized with `[src]=null`
(3) Image loader sets blank/gray background-image, starts loading the placeholder image
(4) Race condition between placeholder-img loading, and CMS returning the `prx:image`
(5) Changes to `[src]=real-image`
(6) Blank/gray background-image, starts loading the real image

Maybe this is an edge case, since CMS usually bundles the `prx:image` in the same json response as the parent story/series/account.  Or maybe the image-loader needs to be smart enough to know if CMS is still loading vs there really isn't an image.  (Null vs undefined?  Or some other explicit attribute?)